lib/autoupdate/rollout/reconciler.go

```diff
@@ -40,12 +41,19 @@ const (
 	defaultStrategy       = update.AgentsStrategyHaltOnError
 	maxConflictRetry      = 3
 
-	defaultGroupName = "default"
-	defaultStartHour = 12
+	defaultGroupName    = "default"
+	defaultCMCGroupName = defaultGroupName + "-cmc"
+	defaultStartHour    = 12
 )
 
 var (
 	defaultUpdateDays = []string{"Mon", "Tue", "Wed", "Thu"}
+	defaultGroup      = &autoupdate.AgentAutoUpdateGroup{
+		Name:      defaultGroupName,
+		Days:      defaultUpdateDays,
+		StartHour: defaultStartHour,
+		WaitDays:  0,
+	}
```

Comment:
As fun as it can be to play with fire, storing and passing around mutable "default" values that are intended to be immutable is a bit spooky. Can we replace the vars with functions that return brand new values every time?